vm-systemd/qubes-qrexec-agent.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Qubes remote exec agent
-After=qubes-dvm.service
+After=qubes-dvm.service xendriverdomain.service
```

Comment:
@marmarek: Hmm, I always think of qrexec-agent having started as "now the VM is in a reasonable state for other VMs or dom0 to communicate with it", e.g. to attach a block device to it. Sort of like basic.target for Qubes VMs...